viewing_party/party.py

```diff
@@ -0,0 +1,199 @@
+from statistics import mode
+# The first four tests are going to test against this function:
+def create_movie(title, genre, rating):
+    """
+    This function takes title, genre and ratings as parameters and returns 
+    the input arguments as values in a dictionary. 
+    """
+    new_movie = {}
+    if title and genre and rating:
+        new_movie["title"] = title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+        return new_movie
```

Comment:
I recommend creating dictionaries in place when possible:

```suggestion
    if title and genre and rating:
        return {
            "title": title,
            "genre": genre,
            "rating": rating
        }
```